manifests/init.pp

```diff
@@ -113,7 +113,7 @@
 
   if $firewall {
     include 'vsftpd::config::firewall'
-    Class['vsftpd::config::firewall'] -> Class['vsftpd::service']
+    Class['vsftpd::service'] -> Class['vsftpd::config::firewall']
```

Comment:
haha, fun fun

Reply:
is it because of the manifest ordering working it out that `bundle exec rake spec` passes, but when I run `bundle exec rspec spec/classes/init_spec.rb` it fails on dependency cycles?